etc/picongpu/juwels-jsc/batch_picongpu.profile.example

```diff
@@ -10,15 +10,16 @@ export MY_MAIL="someone@example.com"
 export MY_NAME="$(whoami) <$MY_MAIL>"
 
 # Project Information ######################################## (edit this line)
-#   - project account for computing time
+#   - project and account for allocation
 export proj=$(groups | awk '{print $4}')
+export account=$(jutil user projects -n | awk '{print $NF}')
 
 # Text Editor for Tools ###################################### (edit this line)
 #   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
 #export EDITOR="nano"
 
 # Set up environment, including $SCRATCH and $PROJECT
-jutil env activate -p $proj
+jutil env activate -p $proj A $account
```

Comment:
Should this be 
```suggestion
jutil env activate -p $proj -A $account
```
?

Reply:
Thanks for the catch, this is absolutely correct.